index.js

```diff
@@ -0,0 +1,118 @@
+const tripsUrl = 'https://trektravel.herokuapp.com/trips';
+
+const displayStatus = (message) => {
+    $('#status').html(message);
+  };
```

Comment:
This function tries to put messages in some element on the page that has ID `status`, but you've commented that section out in the HTML, which means statuses don't appear.